File: Application/src/main/java/com/shareyourproxy/app/IntroductionActivity.java
package com.shareyourproxy.app;

import android.os.Bundle;

import com.shareyourproxy.app.fragment.MainIntroductionFragment;

/**
 * Introduce a user with a view pager flow.
 */
public class IntroductionActivity extends BaseActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        if (savedInstanceState == null) {
            MainIntroductionFragment mainFragment = MainIntroductionFragment.newInstance();
            getSupportFragmentManager().beginTransaction()
                .replace(android.R.id.content, mainFragment)
                .commit();
        }
    }
}

File: Application/src/main/kotlin/com/shareyourproxy/app/IntroductionActivity.kt
package com.shareyourproxy.app

import android.os.Bundle

import com.shareyourproxy.app.fragment.MainIntroductionFragment

/**
 * Introduce a user with a view pager flow.
 */
internal final class IntroductionActivity : BaseActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        if (savedInstanceState == null) {
            supportFragmentManager.beginTransaction().replace(android.R.id.content, MainIntroductionFragment()).commit()
        }
    }
}
